index.d.ts

```diff
@@ -0,0 +1,107 @@
+declare module "react-native-markdown-view" {
+    import * as React from "react";
+    import { ViewStyle, ImageStyle, TextStyle } from "react-native";
+
+    interface MarkdownStyles {
+        [key: string]: TextStyle | ImageStyle | ViewStyle;
+        blockQuote?: TextStyle;
+        codeBlock?: TextStyle;
+        del?: TextStyle;
+        em?: TextStyle;
+        heading?: TextStyle;
+        heading1?: TextStyle;
+        heading2?: TextStyle;
+        heading3?: TextStyle;
+        heading4?: TextStyle;
+        heading5?: TextStyle;
+        heading6?: TextStyle;
+        hr?: TextStyle;
+        imageWrapper?: ViewStyle;
+        image?: ViewStyle;
+        inlineCode?: TextStyle;
+        link?: TextStyle;
+        list?: ViewStyle;
+        listItem?: ViewStyle;
+        listItemNumber?: ViewStyle;
+        listItemBullet?: ViewStyle;
+        listItemOrderedContent?: ViewStyle;
```

Comment:
I believe these should be `TextStyle`?

Reply:
Agreed, just checked the implementation and I have applied your comments.